utils/utils-lib/src/tracing_telemetry.rs

```diff
@@ -188,55 +89,128 @@ impl TracingTelemetry {
         TracingTelemetry { style, ..self }
     }
     /// Specify whether colour is needed or not.
-    pub fn with_colours(self, colours: bool) -> TracingTelemetry {
-        TracingTelemetry { colours, ..self }
+    pub fn with_colors(self, colors: bool) -> TracingTelemetry {
+        TracingTelemetry { colors, ..self }
     }
+
     /// Initialize the telemetry instance.
-    pub fn init(self) {
+    pub fn init(
+        self,
+        service_name: &str,
+        tracing_tags: Vec<KeyValue>,
+        jaeger: Option<String>,
+        events_url: Option<url::Url>,
+    ) {
+        let stdout = tracing_subscriber::fmt::layer().with_writer(std::io::stdout);
+        let stderr = tracing_subscriber::fmt::layer()
+            .with_writer(std::io::stderr)
+            .with_ansi(self.colors);
+        let init_jaeger: bool;
```

Comment:
This is not needed, tracer Some is the same?

Reply:
Right